wazo_auth_client/commands/token.py

```diff
@@ -86,3 +86,20 @@ def get(self, token, required_acl=None, tenant=None):
             self.raise_from_response(r)
 
         return r.json()['data']
+
+    def list(self, user_uuid=None, **kwargs):
+        if user_uuid is None:
+            raise TypeError('user_uuid must be a string')
```

Comment:
the check and the message mismatch
* check the user_uuid is an instance of str
OR
* change the message to say that user_uuid is not defined